db/schema.rb

```diff
@@ -11,11 +11,10 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20150512134228) do
+ActiveRecord::Schema.define(version: 20150626120818) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
-  enable_extension "pg_trgm"
```

Comment:
i assume rails did that when i ran `rake db:migrate`, because it certainly wasn't intentional.